cluster/manifests/skipper/deployment.yaml

```diff
@@ -1,12 +1,15 @@
-{{ $internal_version := "v0.21.198-1017" }}
-{{ $canary_internal_version := "v0.21.198-1017" }}
+{{/* image-updater-bot detects *image variables so use __ suffux to disable it for main image */}}
+
+{{ $main_image__ := "container-registry.zalando.net/teapot/skipper:v0.21.198-1017" }}
+{{ $canary_image := "container-registry.zalando.net/teapot/skipper:v0.21.198-1017" }}
```

Comment:
@AlexanderYastrebov e2e fails because of Image pull backoff

```
                state:
                  waiting:
                    message: Back-off pulling image "926694233939.dkr.ecr.eu-central-1.amazonaws.com/production_namespace/teapot/skipper:v0.21.198-1017"
                    reason: ImagePullBackOff
```
I believe the name should be `skipper-internal` and not `skipper`

Reply:
:facepalm: Thanks